libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -765,6 +765,20 @@ void AP_MotorsMatrix::setup_motors(motor_frame_class frame_class, motor_frame_ty
                     add_motors(motors, ARRAY_SIZE(motors));
                     break;
                 }
+                case MOTOR_FRAME_TYPE_Y4:
+                    _mav_type = MAV_TYPE_QUADROTOR;
```

Comment:
```suggestion
```
Don't need this it is set at the start of the quad case.

Reply:
removed...thanks